### ISSUE TITLE
How to get Dates when reading a file?

### ISSUE THREAD

**Issue by dmint007** [360EntSecGroup-Skylar/excelize#704]
**Description**
Then how can we fetch the date in the given cell?

When reading from an `xsls` file, a simple column with a hard-coded date `09/09/19`, **GetCellValue()** keeps returning as "43717.0".

I found out this piece of code might be useful `v := excelize.parseTime(22, cellValue)` but is unexported.

I also tried this, but still returns "43717.0".
```
style, _ = xlsx.NewStyle(`{"number_format":15}`)
xlsx.SetCellStyle("Sheet1", "E7", "E7", style)
e7 := xlsx.GetCellValue("Sheet1", "E7")
println(e7) // 29-Jun-17
```

**Steps to reproduce the issue:**
1. Read file
2. Hard-code date
3. ???
4. Profit

**Describe the results you received:**
"43717.0"

**Describe the results you expected:**
I expected a string or a `time.Time` object

**Output of `go version`:**

```text
go version go1.13.6 darwin/amd64
```

**Excelize version or commit ID:**

```text
v2.3.0
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
Google Docs Sheets

![image](https://user-images.githubusercontent.com/52902269/92982459-e0e0f400-f463-11ea-8192-615d0d08e7f8.png)

**Comment by ramacatur** [comment]
Please check in this issue #73

**Comment by dmint007** [comment]
> Please check in this issue #73

I did, but none of the solutions are not working for me. My temp solution was to copy `excelize.parseTime()` into my project.
Wish it was public.

**Comment by ramacatur** [comment]
I use this solution and it works

```
// note: this is off by two days on the real epoch (1/1/1900) because
// - the days are 1 indexed so 1/1/1900 is 1 not 0
// - Excel pretends that Feb 29, 1900 existed even though it did not
// The following function will fail for dates before March 1st 1900
// Before that date the Julian calendar was used so a conversion would be necessary
var excelEpoch = time.Date(1899, time.December, 30, 0, 0, 0, 0, time.UTC)

func excelDateToDate(excelDate string) time.Time {
	var days, _ = strconv.Atoi(excelDate)
	return excelEpoch.Add(time.Second * time.Duration(days*86400))
}
```

**Comment by AngangGuo** [comment]
My Microsoft Excel file use format like "9/15/2020" and excelize get it like "9/15/20". 
Later on I change some other data in LibraOffice, the date format changes as 09-15-2020. This time excelize read it as 44089.
I applied the column style "of.SetColStyle(name,"B",14)" and hope it will be right, but with no success.
You never know how to compare this column data with a date.

**Comment by xuri** [comment]
Resolved by PR #703, if you have any questions, please let me know.

**Comment by fatindeed** [comment]
when upgrade v2.3.1 to v2.3.2
`f.GetCellValue()` will return a date-time string, like `Jan 1`. It's hard to be converted to time.
So you must set cell style to `0`, and get cell value again. Then it will return the float value and you can convert it to time.
```
```
err := f.SetCellStyle("sheet1", "A1", "A1", 0)
if err != nil {
	...
}
label, err := f.GetCellValue("sheet1", "A1")
if err != nil {
	...
}
f, err := strconv.ParseFloat(label, 64)
if err != nil {
	...
}
t, err = excelize.ExcelDateToTime(f, false)
if err != nil {
	...
}
```
